cmd/cape/cmd/key.go

```diff
@@ -83,3 +90,6 @@ func GetKeyRequest(pcrSlice []string, token string) (sdk.KeyRequest, error) {
 		PcrSlice:     pcrSlice,
 	}, nil
 }
+
+var authTokenFunc = getAuthToken
```

Comment:
I think we should already have a global function for this. Think its called `authToken`.